packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -149,16 +149,33 @@ Future<void> main() async {
 }
 ''';
     } else {
+
+// TODO: Add the FlutterLoader code to the if-case above (with plugins)!
+
       contents = '''
 // @dart=${languageVersion.major}.${languageVersion.minor}
 
 import 'dart:ui' as ui;
 
+import 'package:flutter_web_plugins/flutter_loader.dart';
```

Comment:
IIUC, every app now has to add a dependency on `flutter_web_plugins`? Can we add the dependency in `packages/flutter/pubspec.yaml`?

Reply:
I think you're right. It seems that currently (when there's plugins) we rely on the transient dependency of the plugins for this `import`, but now we might need to add this to every app:

https://github.com/flutter/flutter/blob/65c6e88223b0a2156c28b2cf44a70592bf8bc284/packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl#L13-L16

This is a shame, because it makes this change non-backwards compatible for apps without plugins :(

Reply:
Confirmed this breaks.

```
Error: Couldn't resolve the package 'flutter_web_plugins' in 'package:flutter_web_plugins/flutter_loader.dart'.
org-dartlang-app:/web_entrypoint.dart:9:8: Error: Not found: 'package:flutter_web_plugins/flutter_loader.dart'
import 'package:flutter_web_plugins/flutter_loader.dart';
       ^
```

Also confirmed that `dependency_overrides` gets a package resolved without it showing up in its dependencies from pubviz (somewhat dangerous)

Reply:
We're not relying on flutter_web_plugins for this first part of the PR, at least.